packages/nouns-webapp/src/utils/etherscan.ts

```diff
@@ -33,15 +34,14 @@ export const buildEtherscanHoldingsLink = (address: string): string => {
 };
 
 const getApiBaseURL = (network: number) => {
-  switch (network) {
-    case sepolia.id:
-      return 'https://api-sepolia.etherscan.io/';
-    default:
-      return 'https://api.etherscan.io/';
+  if (network === sepolia.id) {
+    return 'https://api-sepolia.etherscan.io/';
```

Comment:
could use `defaultChain.blockExplorers.default.apiUrl`

Reply:
It’s done